src/region.js

```diff
@@ -84,9 +84,9 @@ const Region = MarionetteObject.extend({
     }
   },
 
-  _attachView(view) {
+  _attachView(view, {replaceElement} = {}) {
     const shouldTriggerAttach = !view._isAttached && isNodeAttached(this.el);
-    const shouldReplaceEl = !!this.getOption('replaceElement');
+    const shouldReplaceEl = !!replaceElement || !!this.getOption('replaceElement');
```

Comment:
Yeah, you're right. My mistake - I was caught up on trying to use the same destructuring syntax and missed the obvious logic error.

I think I prefer `replaceElement === true` rather than `replaceElement !== false` to avoid using double-negative. Should be equivalent result, though, yeah?